01 - JavaScript Drum Kit/index-START.html

```diff
@@ -1,66 +1,122 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
```

Comment:
great work, G. Please find my (minor) comments below.

Reply:
thanks Tom! this is real fun 👍